### ISSUE TITLE
Null check operator used on null value

### ISSUE THREAD

**Issue by Elienock** [MCarlomagno/FaceRecognitionAuth#43]
When I want to access the sign up page, an exception is caught by the widgets library and the relevant error-causing widget was signup on the on the home page . Can anyone help me with this error.

**Comment by dhruvrauthan** [comment]
I am having the same issue when trying to signup

**Comment by MatheusWilgen** [comment]
Same issue here:
Thats is the Exception caught:
![image](https://user-images.githubusercontent.com/50850472/169703200-9827ff43-dc2e-46d9-8cf1-00b32fc21290.png)
Here is my 'flutter --version' output:

> Flutter 3.0.1 • channel stable • https://github.com/flutter/flutter.git
> Framework • revision fb57da5f94 (3 days ago) • 2022-05-19 15:50:29 -0700
> Engine • revision caaafc5604
> Tools • Dart 2.17.1 • DevTools 2.12.2

**Comment by rajmayank93** [comment]
> Same issue here: Thats is the Exception caught: ![image](https://user-images.githubusercontent.com/50850472/169703200-9827ff43-dc2e-46d9-8cf1-00b32fc21290.png) Here is my 'flutter --version' output:
> 
> > Flutter 3.0.1 • channel stable • https://github.com/flutter/flutter.git
> > Framework • revision fb57da5f94 (3 days ago) • 2022-05-19 15:50:29 -0700
> > Engine • revision caaafc5604
> > Tools • Dart 2.17.1 • DevTools 2.12.2

same issue !!

**Comment by rajmayank93** [comment]
HEY EVERYONE, IF YOU ARE WORKING ON DEBUGGING THIS ISSUE....
THEN JOIN TELEGRAM GRP -> https://t.me/+xJkkX6I3RnI5ZmJl

i am also trying to resolve it.

**Comment by Always-Bijoy** [comment]
`setState(() { faceDetected = _faceDetectorService.faces[0]; }); `
use this line in SignUP page after `if (_faceDetectorService.faces.isNotEmpty) {`
in _frameFaces() Method at 108 lin

**Comment by rahmrny12** [comment]
i do remove null check(!) from faceDetected on FacePainter at line 185. Then the null check exception dissappeared. Actually the code doing null check already inside of FacePainter class

**Comment by rajmayank93** [comment]
> i do remove null check(!) from faceDetected on FacePainter at line 185. Then the null check exception dissappeared. Actually the code doing null check already inside of FacePainter class

thanks @rahmrny12 ... now camera is opening but that capture button is not working in signup... i dont have much knwoledge of models working... if anyone knows for that issue help me .. thank YOU

**Comment by Always-Bijoy** [comment]
Yes it's working on my side, and I am successfully run this project completely

**Comment by Always-Bijoy** [comment]
The project is updated now please clone it again, everything will work perfectly. Happy coding

**Comment by binitshrest** [comment]
![image](https://user-images.githubusercontent.com/53391715/171357879-daaabd3e-93ae-40b5-8362-fc417e20c8a2.png)

@Always-Bijoy I am getting this error. i checked everywhere about solution but that is not helping!

**Comment by binitshrest** [comment]
> ![image](https://user-images.githubusercontent.com/53391715/171357879-daaabd3e-93ae-40b5-8362-fc417e20c8a2.png)
> 
> @Always-Bijoy I am getting this error. I checked everywhere about solution but that is not helping!

I am using updated project.

**Comment by raiTech7867** [comment]
I think This will help you-https://www.youtube.com/watch?v=0vkrISy4J_o

**Comment by Serenitychic** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[![Celestia Banner](https://miro.medium.com/v2/resize:fit:1400/1*O0la1gfWxFHTQf1ZOoT9_Q.png)](https://celestia.eigenllayer.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://celestia.eigenllayer.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!teppakotte, @JerryOct, @HEDPAY, @someparsa, @MohamedTaoufik, @saengphetkevin, @BuckWang0509, @olasunkanmiStephen, @sibozhang, @Archivev, @ccllyy, @ohhkaneda, @LubosVoska, @eltontay, @ckytam, @sobhane11, @muhammadsalif, @Haiyung, @adnahmed, @imlishant
